src/components/Widget/components/Conversation/components/Sender/components/Suggestions/index.js

```diff
@@ -9,37 +9,54 @@ import { connect } from 'react-redux';
 
 import { setSelectedSuggestion } from '../../../../../../../../store/actions';
 
+const reactStringReplace = require('react-string-replace');
+
+
 
 class SuggestionsList extends PureComponent {
 
   constructor(props) {
     super(props);
-    this.state = { hasOverflow: false, endScroll: false, startScroll: true };
+    this.state = { hasOverflow: false, endScroll: false, startScroll: true, scrollDistance: 300 };
     this.suggestionsListRef = React.createRef();
-    this.scrollDistance = 250;
+  }
+
+  hasOverflow = (element) => {
+    return element.offsetHeight < element.scrollHeight || element.offsetWidth < element.scrollWidth
   }
 
   componentDidMount() {
     const element = this.suggestionsListRef.current;
-    let hasOverflow = element.offsetHeight < element.scrollHeight || element.offsetWidth < element.scrollWidth;
+    let hasOverflow = this.hasOverflow(element);
     this.setState({ hasOverflow })
+    // the + 10 is to add the margin from the prev and next buttons since offsetWidth doesn't include that
+    this.setState({ scrollDistance: element.offsetWidth + 10 })
   }
 
   left = () => {
-    this.suggestionsListRef.current.scrollLeft -= this.scrollDistance;
+    const element = this.suggestionsListRef.current;
+    // the + 10 is to add the margin from the prev and next buttons since offsetWidth doesn't include that
+    element.scrollLeft -= this.state.scrollDistance + 10;
+    this.setState({ scrollDistance: element.offsetWidth + 10 })
   };
   right = () => {
-    this.suggestionsListRef.current.scrollLeft += this.scrollDistance;
+    const element = this.suggestionsListRef.current;
+    // the + 10 is to add the margin from the prev and next buttons since offsetWidth doesn't include that
+    element.scrollLeft += this.state.scrollDistance + 10;
```

Comment:
The functions left and right appear to be the same, on function right() should be element.scrollRight ?
maybe it's better create just one function and get element.scroll by params